peer/peer.go

```diff
@@ -80,6 +82,10 @@ func (p *Peer) Gossip(ctx context.Context, contentID []byte, subnet *id.Hash) {
 	p.gossiper.Gossip(ctx, contentID, subnet)
 }
 
+func (p *Peer) PeerDiscovery(ctx context.Context) {
```

Comment:
Methods are conventionally named as verbs. This should be `DiscoverPeers` (likewise for the method on the actual client object too).